ballerina/http_types.bal

```diff
@@ -119,3 +119,11 @@ public type Links record {|
     # Array of available links
     Link[] links;
 |};
+
+# Represents the parsed header value details
+public type HeaderValue record {|
+    # The header value
+    string value;
+    # Map of header parameters
+    map<string> params;
+|};
```

Comment:
Shall we add standard doc-comments ?
```suggestion
# Represents the parsed header value details
#
# + value - The header value
# + params - Map of header parameters   
public type HeaderValue record {|
    string value;
    map<string> params;
|};
```

Reply:
I think what @chamil321 has added works too.